libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -185,6 +212,42 @@ void AP_AHRS::add_trim(float roll_in_radians, float pitch_in_radians, bool save_
     }
 }
 
+// Set the board mounting orientation, may be called while disarmed
+void AP_AHRS::set_orientation()
+{
+    enum Rotation orientation = (enum Rotation)_board_orientation.get();
+    if (orientation != ROTATION_CUSTOM) {
+        _ins.set_board_orientation(orientation);
+        if (_compass != nullptr) {
+            _compass->set_board_orientation(orientation);
+        }
+    } else {
+        float xrad = DEG_TO_RAD * _rot_roll;
+        float yrad = DEG_TO_RAD * _rot_pitch;
+        float zrad = DEG_TO_RAD * _rot_yaw;
+        Matrix3f rotationx = Matrix3f(
```

Comment:
is there some reason we can't use the existing euler to matrix functions?

Reply:
where can I find these?